src/commcare_cloud/ansible/roles/deploy_hq/tasks/update_translations.yml

```
@@ -7,3 +7,8 @@
   loop:
     - update_django_locales
     - compilemessages -v 0
+
+- name: Compile Django JS Translations
+  command:
+    cmd: ./manage.py compilejsi18n
+    chdir: '{{ code_source }}'
```

Could `compilejsi18n` be added to the previous task loop?
```suggestion
    - compilejsi18n
```

---

I don't think there is any reason not to keep in the loop. It should be essentially be same. I will make the changes.